xp/identityserver/Dockerfile

```diff
@@ -31,12 +31,14 @@ COPY --from=prepare /Files/Config /Files/Config/
 ADD scripts /Scripts
 
 ENV IDENTITY_CERT_PATH "C:\\Files\\identity.pfx"
+ENV ROOT_CERT_PATH "C:\\Files\\root.pfx"
 
  #Instal donet core windows hosting
 RUN choco install -y --params="Quiet" dotnetcore-windowshosting; 
 
  # Trust Self signed certificates & certificate
-RUN /Scripts/Import-Certificate.ps1 -certificateFile $Env:IDENTITY_CERT_PATH -secret 'secret' -storeName 'Root' -storeLocation 'LocalMachine'; `
+RUN /Scripts/Import-Certificate.ps1 -certificateFile $Env:ROOT_CERT_PATH -secret 'secret' -storeName 'Root' -storeLocation 'LocalMachine'; `
+    /Scripts/Import-Certificate.ps1 -certificateFile $Env:ROOT_CERT_PATH -secret 'secret' -storeName 'My' -storeLocation 'LocalMachine'; `
```

Comment:
in My & Root?

Reply:
Yes, normally it should only go to root, but since we use it to dynamically create and sign certificates during boot, it needs to be in my. It will fail to sign the certificate otherwise.